supervisor_types.go

```diff
@@ -415,7 +429,7 @@ type IngestionSpecOptions func(*InputIngestionSpec)
 func SetType(stype string) IngestionSpecOptions {
 	return func(spec *InputIngestionSpec) {
 		if stype != "" {
-			spec.IOConfig.Type = stype
+			spec.Type = stype
```

Comment:
QQ: What does this change mean?

Reply:
This change sets type of the ingestion specification, as `spec.Type` is the primary point where the ingestion specification type should be specified. https://druid.apache.org/docs/latest/ingestion/#ingestion-methods describes the values it can take. Type of the IOConfig ingestion spec is rudimentary and does not need to be set.